pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
Having a host mount scc is ok in general.

On Nov 9, 2015, at 11:30 AM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/cmd/server/kubernetes/master.go
<https://github.com/openshift/origin/pull/5792#discussion_r44319018>:

> @@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
>  }
>
>  func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
> +	privilged := true

if we set this to privileged, there is no point to the hostmount SCC...
recycler pods will fail until the recycler service account is added to the
privileged SCC

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5792/files#r44319018>.